src/Dialect/Mlir/DialectBuilder.cpp

```diff
@@ -2073,6 +2073,29 @@ void VectorBuilder::multiReduction(ArrayRef<Value> inputVecArray,
   }
 }
 
+Value VectorBuilder::extractElement(Value vector, int64_t index) const {
```

Comment:
Added for experiments on how to get better performance, currently not in use but were verified to work during experiments associated with the PR